src/panels/KeybindsPanel/KeybindsPanel.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Badge,
+  Code,
+  Container,
+  Divider,
+  Grid,
+  Group,
+  Paper,
+  Table,
+  Text,
+  Title
+} from '@mantine/core';
+
+import { Action } from '@/types/types';
+
+import { FullKeyboard } from './FullKeyboard/FullKeyboard';
+import { KeybindButtons } from './Keybind';
+
+export function KeyBindsPanel() {
+  const [selectedActions, setSelectedActions] = React.useState<Action[]>([]);
+  const [activeModifiers, setActiveModifiers] = React.useState<string[]>([]);
+  const [selectedKey, setSelectedKey] = React.useState<string>('');
+
+  const hasSelectedKeys = selectedKey === '' && activeModifiers.length === 0;
+  return (
+    <Container maw={'none'}>
+      <FullKeyboard
+        setSelectedActions={setSelectedActions}
+        setActiveModifiers={setActiveModifiers}
+        setSelectedKey={setSelectedKey}
+        selectedKey={selectedKey}
+        activeModifiers={activeModifiers}
+      />{' '}
+      <Group my={'md'}>
+        <Title order={3}>Selected keybind:</Title>
+        <KeybindButtons selectedKey={selectedKey} modifiers={activeModifiers} />
+      </Group>
+      <Divider />
+      <Title order={3} my={'md'}>
```

Comment:
I had comments from Emma on this, we should decide how we want to deal properly with lower-order titles when there aren't any higher ones. Something to do with information flow for screen readers? @WeirdRubberDuck

Reply:
Yes, I think we should stick with a title hierarchy that reflects the information hierarchy of the page, and be consistent in all out panels. The hierarchy of the titles conveys a lot of information to screen reader users. 

https://atendesigngroup.com/articles/guide-accessible-heading-hierarchy

An important note is that `h3` titles like this should not exist without a h2 title, for context (see link above).  So this code here is an issue. 

One alternative could be that we consider the tab name the "h2" title for sighted users, and add visible hidden titles of order 2 for screen reader users.   

Let's discuss IRL perhaps and come up with a scheme that we think will work well for us?

Reply:
Either way, importantly, the Title order should not be used to set a desired size of the title - it should reflect the hierarchy of the site